product_vision_and_principles.md

```diff
@@ -0,0 +1,65 @@
+	2. "Change agents" = people actively trying to make the organisational system (or ecosystem) work better.
+- Our theory of change is basically as follows:
+	1. We provide **stakeholders** with a tool that helps them use maps and linked data to do everyday things much more easily compared to the corporate tools they're used to
+	2. The more they use the tool, the more shared meaning we create across organisational boundaries, because the structure of the data and the visual language of the maps all follow a consistent ontology, and the data and maps stay close to reality through the activity of a Shared Meaning Network
+	3. We help **change agents** take advantage of this shared meaning by designing custom interactions on the SMP to support systems interventions (e.g. telling stories, asking questions, visualising patterns ...)
+	4. As stakeholders become more engaged with the platform, they become "change agents" themselves, using more of the change agent tools to support interventions of their own, thus creating a virtuous cycle in which more shared meaning is generated across larger ecoysystems, unlocking the potential for more significant system changes
+- We will continue to run a smaller proportion of projects outside of these long term partnerships, but they will still be based on the same theory of change, i.e. generating shared meaning from consistent base maps, on which interactions can be designed that are tailored to the intervention need 
+
+The principle is that this theory of change needs to be reflected in our design decisions, particularly the distinction between "core platform features" for stakeholders and "customisable tools" for change agents, as described in the next principle.
+
+## Guiding Principle 2: Separate core from extensible functionality
+
+- Core platform features:
+	- should be at least as simple and intuitive to use for non-experts as Google Maps
+	- should be optimised, stable, and not change very much.
+	- have two primary use cases:
+		1. to help new folks understand how their organisational system/eco-system works and where they fit into it
+		2. to help everyday folks find organisational (as opposed to operational) information that would otherwise be hard to track down because it lives on a bunch of different corporate systems (e.g. Who owns this process? When does the contract for this IT system expire? What does this acronym stand for? Who else does this activity? etc.)
+	- should cover the following interactions and nothing else: search, map switching, map exploration, item selection, link navigation
+	- should also include an "advanced" mode for expert users who want to perform more advanced filters / queries directly on the model. This should include list mode, graph mode (?), filter tools, advanced search, detail level control. How this integrates with the basic experience is an ongoing design challenge; whatever choices we make must not jeopardise the simplicity of the experience for new users.
```

Comment:
I don't like the "advance mode". If this is only for internal use, ok. But google maps doesn't have such mode. I prefer to have good UX design and help page with some nice videos maybe instead to hide the functionalities under some modes.